src/views/accounts/index.js

```diff
@@ -55,6 +55,7 @@ export const Accounts = ({ navigation }) => {
         modalButtonsContainer: {
             flexDirection: "row",
             justifyContent: "center",
+            padding: 5,
```

Comment:
See above.

Reply:
This is not needed, I'm going to undo this.